### PR TITLE
feat: Add new component, grants data visualisation prototype with Rea…

### PR DESCRIPTION
**Pull request by KoalaMango**
…ct-Vis, fixes #STOR-104

### CODE REVIEW

File: src/components/GrantsInfographics/GrantsInfographics.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        const chartData1 = [];
+        const chartData2 = [];
+
+        grantData.issue.map((data) => {
+          barData.push({ x: data.key, y: data.count });
+          return barData;
+        });
+
+        grantData.country_name.filter((r) => { return r.key === 'UNITED KINGDOM'; }).map((data) => {
+          const projects = Math.trunc((data.count/projectTotal)*100);
```
Comment (lebaz20) [inline src/components/GrantsInfographics/GrantsInfographics.js]: I think trunc all over the items mayblead to having total not equal to 100%, probably less

File: src/components/GrantsInfographics/GrantsInfographics.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        const barData = [];
+        const chartData1 = [];
+        const chartData2 = [];
+
+        grantData.issue.map((data) => {
+          barData.push({ x: data.key, y: data.count });
+          return barData;
+        });
+
+        grantData.country_name.filter((r) => { return r.key === 'UNITED KINGDOM'; }).map((data) => {
```
Comment (lebaz20) [inline src/components/GrantsInfographics/GrantsInfographics.js]: Why filtering here on uk ?

File: src/components/GrantsInfographics/GrantsInfographics.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+          const projects = Math.trunc((data.count/projectTotal)*100);
+          const awarded = Math.trunc((data.total_awarded/awardedTotal)*100);
+
+          chartData1.push(
+            { theta: projects/10, color: RandomColor({ hue: 'blue', luminosity: 'light' }), label: 'United Kingdom', subLabel: `${projects}%` },
```
Comment (lebaz20) [inline src/components/GrantsInfographics/GrantsInfographics.js]: I don’t understand dividing by 10 here and the other item (10- projects/10) ?

**Comment by KoalaMango** [comment]
@lebaz20 I want to display pie chart with the percentage of UK and International in
- amount of awarded 
- active projects

Data reference:
- grant api: https://1kfs7evxca.execute-api.eu-west-1.amazonaws.com/beta/grants
- react-vis pie chart data structure: http://uber.github.io/react-vis/documentation/other-charts/radial-chart

Thought process: 
1. **Get data:** Grant API JSON already has total numbers of awarded and active projects. All I need is to get UK numbers. Then `Total - UK = International` and get percentage of both.
2. **Process data:** I maybe wrong but I assume the total of `theta` is 10, which represents  percentage. Instead of between 1 - 100, the number should between 1 - 10. Hence, the `dividing by 10` and `(10 - projects/10)`. I probably could do it better with less repeating vars.

**Pull request merged**